### ISSUE TITLE
Setting Zoom 

### ISSUE THREAD

**Issue by KateNegrienko** [ihor-zinchenko/ngx-apple-maps#10]
Hi,

How do I set the zoom during map initialization? 

Thanks!

**Comment by ihor-zinchenko** [comment]
@KateNegrienko hi! Unfortunatelly you can't set zoomlevel before the map API will initialize, after initialization API you will get an object with all posible methods to work with map, for example after
onLoaded(e) {
  this.map = e;
}

you can use this.map.zoom = 15 (any number)

**Comment by KateNegrienko** [comment]
I tried to do it and in console I don't see errors, but nothing changed 
with zoom = 20;
![image](https://user-images.githubusercontent.com/13833386/105710314-99916680-5f1f-11eb-8da3-565f48a08dbe.png)
with zoom = 50;
![image](https://user-images.githubusercontent.com/13833386/105710384-b5950800-5f1f-11eb-8a6b-d3661cadbd6e.png)

```
onLoaded(e) {
    this.map = e;
    this.map.zoom = 50;
  }

```

**Comment by danhadzami** [comment]
@KateNegrienko Hi, I had a similar issue. I forgot to put this into my HTML setup:

`(onLoaded)="onLoaded($event)"`

So, it will look something like this:

```
        <ngx-apple-maps [options]="options" (onLoaded)="onLoaded($event)"></ngx-apple-maps>
```

**Comment by webtimo-de** [comment]
Hello guys, I have a solution for the zoom problem. I also brought this project to Angular 16 and Ivy. You can find this at `ngx-apple-mapkit`.

```shell
npm install ngx-apple-mapkit
```

I have a [workaround for zoom in the documentation](https://github.com/webtimo-de/ngx-apple-mapkit/blob/e414b10a58172fb92f556c88eb1cae2f38441b4b/README.md#zoom). I hope I can help you with this, even if it's been 2 years now.

**Comment by ihor-zinchenko** [comment]
"Fixed in v1.0.0. Older versions are no longer supported. It will be released soon and will be public."
